pkg/datasources/stages.go

```diff
@@ -56,42 +57,55 @@ var stagesSchema = map[string]*schema.Schema{
 
 func Stages() *schema.Resource {
 	return &schema.Resource{
-		Read:   ReadStages,
-		Schema: stagesSchema,
+		ReadContext: ReadStages,
```

Comment:
i didnt know about this ReadContext function. we could actually use this for all resources, as we are using context.Background()

Reply:
Yeah, this is one of the points I added to our discussion on our resource-designing conventions. I'm using it because Read is deprecated and ReadContext or ReadWithoutTimeout should be used instead.